.github/workflows/3rd-party-licenses.yaml

```diff
@@ -0,0 +1,107 @@
+# Run it locally with act
+#  1. Install act:
+#     `brew install act`
+#  2. Create a .secret file with the following content:
+#     `GITHUB_TOKEN=your_github_token`
+#  3. Create a pull_request.json file in case of a pull request with the following content:
+#     `{"pull_request": {"number": <PR number>, "head": {"ref": "<PR branch name>", "sha": "PR commit sha"}, "base": {"ref": "main"}}, "repository": {"name": "greenhouse", "owner": {"login": "cloudoperators"}}}`
+#  4. Run the following command:
+# `act pull_request --container-architecture linux/amd64 -P default=catthehacker/ubuntu:act-latest  -j check-3rd-party-licenses -e act_pull_request.json`
+
+name: 3rd Party License Checker
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
```

Comment:
```suggestion
    types: [opened, synchronize, reopened]
    paths:
      - 'ui/**'
```

Reply:
I would add a filter here to run it only when `ui` elements changed

Reply:
yes, this is a good idea